src/marine/cryosat_ice2ioda.py

```diff
@@ -24,13 +24,12 @@
 sys.path.append(str(IODA_CONV_PATH.resolve()))
 
 from orddicts import DefaultOrderedDict
-import ioda_conv_ncio as iconv
+import ioda_conv_engines as iconv
 
 
 class Observation(object):
 
-    def __init__(self, filename, thin, date, writer):
-        print(date)
+    def __init__(self, filename, thin, date):
 
         if os.path.exists("cryosat_nc4classic.nc"):
             os.remove("cryosat_nc4classic.nc")
```

Comment:
Is there any reason why we are repeatedly removing "cryosat_nc4classic.nc"?

Reply:
I agree, I removed these two lines.